docs/_posts/josejuanmartinez/2022-01-17-ner_deid_subentity_roberta_es.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: model
+title: Detect PHI for Deidentification purposes (Spanish)
+author: John Snow Labs
+name: ner_deid_subentity_roberta
+date: 2022-01-17
+tags: [deid, es, licensed]
+task: De-identification
+language: es
+edition: Spark NLP for Healthcare 3.3.4
+spark_version: 3.0
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+Named Entity recognition annotator allows for a generic model to be trained by utilizing a deep learning architecture (Char CNNs - BiLSTM - CRF - word embeddings) inspired on a former state of the art model for NER: Chiu & Nicols, Named Entity Recognition with Bidirectional LSTM,CNN. 
+
+Deidentification NER (Spanish) is a Named Entity Recognition model that annotates text to find protected health information that may need to be de-identified. It detects 13 entities. This NERmodel is trained with a combination of custom datasets, Spanish 2002 conLL, MeddoProf dataset and several data augmentation mechanisms
```

Comment:
This NER model is trained with a combination of custom datasets, Spanish 2002 conLL, MeddoProf dataset and several data augmentation mechanisms.